### PR TITLE
Feature/hp 48/cost calculating

### PR DESCRIPTION
**Pull request by bbrands02**

### CODE REVIEW

File: src/Service/PaymentService.php [hunk 1]
```diff
@@ -102,6 +105,109 @@ public function checkSourceAuth(Source $source): bool
...
+
+
+    /**
+     * Get product prices from this marriage.
+     *
+     * @param array $huwelijk
+     *
+     * @return array $productPrices
+     */
+    public function getProductPrices(array $huwelijk): array
```
Comment (WilcoLouwerse) [inline src/Service/PaymentService.php:149]: Not so clean, maybe add todo: "split into functions, nesting level is higher than 3"?
Or just clean it up.

File: src/Service/PaymentService.php [hunk 2]
```diff
@@ -102,6 +105,109 @@ public function checkSourceAuth(Source $source): bool
...
+
+    }//end getProductPrice()
+
+
+    /**
+     * Get product prices from this marriage.
+     *
+     * @param array $huwelijk
+     *
+     * @return array $productPrices
```
Comment (WilcoLouwerse) [inline src/Service/PaymentService.php:147]: Missing return doc

File: src/Service/PaymentService.php [hunk 3]
```diff
@@ -102,6 +105,109 @@ public function checkSourceAuth(Source $source): bool
...
+
+        return null;
+
+    }//end getProductPrice()
+
+
+    /**
+     * Get product prices from this marriage.
+     *
+     * @param array $huwelijk
```
Comment (WilcoLouwerse) [inline src/Service/PaymentService.php:145]: missing param doc

File: src/Service/PaymentService.php (hunk 2 above)
Comment (WilcoLouwerse) [reply src/Service/PaymentService.php:147]: Same for other functions

**Review by WilcoLouwerse — CHANGES_REQUESTED** [review]

**Review by WilcoLouwerse — APPROVED** [review]

**Comment by WilcoLouwerse** [comment]
Approved, but only merge if @smisidjan approves this as well

**Review by smisidjan — APPROVED** [review]

**Pull request merged**